arch/arm/adler32_neon.c

```diff
@@ -18,7 +18,11 @@
  */
 #include "adler32_neon.h"
 #if defined(__ARM_NEON__) || defined(__ARM_NEON)
-#include <arm_neon.h>
+#ifdef _M_ARM64
+# include <arm64_neon.h>
```

Comment:
Should be two spaces between # and include.

Reply:
Indentation in these files doesn't conform to our own guidelines yet... Whole file needs to be reformatted... I can change the 1 space indentation to 2 space indentation, but that doesn't completely fix the formatting...